### PR TITLE
Add structure test for debugger jvm args

### PR DESCRIPTION
**Pull request by aslo**
This test ensures that the correct JVM args for the debugger are set.

### CODE REVIEW

File: jetty9/src/test/workspace/jetty-setup-debugger.bash [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+EXPECTED_DBG_AGENT='-agentpath:/opt/cdbg/cdbg_java_agent.so=--log_dir=/var/log/app_engine,--alsologtostderr=true,--cdbg_extra_class_path=/var/lib/jetty/webapps/root/WEB-INF/classes:/var/lib/jetty/webapps/root/WEB-INF/lib'
```
Comment (joaoandremartins) [inline jetty9/src/test/workspace/jetty-setup-debugger.bash]: Should we test against `$JETTY_BASE` rather than `/var/lib/jetty`?

**Comment by aslo** [comment]
@joaoandremartins Updated to use jetty_base. If you're happy with this, feel free to merge whenever you like.

File: jetty9/src/test/workspace/jetty-setup-debugger.bash [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+EXPECTED_DBG_AGENT="-agentpath:/opt/cdbg/cdbg_java_agent.so=--log_dir=/var/log/app_engine,--alsologtostderr=true,--cdbg_extra_class_path=${JETTY_BASE}/webapps/root/WEB-INF/classes:${JETTY_BASE}/webapps/root/WEB-INF/lib"
+ACTUAL_DBG_AGENT="$(export GAE_INSTANCE=instance; /docker-entrypoint.bash env | grep DBG_AGENT | cut -d '=' -f 1 --complement)"
+
+if [ $ACTUAL_DBG_AGENT != "$EXPECTED_DBG_AGENT" ]; then
```
Comment (joaoandremartins) [inline jetty9/src/test/workspace/jetty-setup-debugger.bash]: Does $ACTUAL_CBG_AGENT require quotes too?

Comment (aslo) [inline jetty9/src/test/workspace/jetty-setup-debugger.bash]: Not required but I think it's "good bash" to have them. I'll add them.

**Pull request merged**